### PR TITLE
added mordred

### PR DESCRIPTION
**Pull request by aclyde11**
I added Mordred descriptors. In this branch because of change Harry made. Has been tested and works. Files are on FTP.

### CODE REVIEW

**Comment by Tipizen** [comment]
Excellent.

File: Pilot1/Uno/uno_data.py [hunk 1]
```diff
@@ -281,6 +281,38 @@ def load_drug_data(ncols=None, scaling='std', imputing='mean', dropna=None, add_
...
+    df = pd.read_csv(path, engine='c', sep='\t', na_values=['na', '-', ''])
+    df.iloc[:, 1:] = df.iloc[:, 1:].apply(pd.to_numeric, errors='coerce')
+    df.iloc[:, 1:] = df.iloc[:, 1:].astype(np.float32)
+
+    df1 = pd.DataFrame(df.loc[:, 'DRUG'])
+    df1.rename(columns={'DRUG': 'Drug'}, inplace=True)
+
+    df2 = df.drop('DRUG', 1)
+    if add_prefix:
+        df2 = df2.add_prefix('dragon7.')
```
Comment (hyoo) [inline Pilot1/Uno/uno_data.py]: Is this correct prefix name?

File: Pilot1/Uno/uno_data.py [hunk 2]
```diff
@@ -281,6 +281,38 @@ def load_drug_data(ncols=None, scaling='std', imputing='mean', dropna=None, add_
...
+
+    df2 = impute_and_scale(df2, scaling, imputing)
+
+    df_desc = pd.concat([df1, df2], axis=1)
+
+    df1 = pd.DataFrame(df_desc.loc[:, 'Drug'])
+    df2 = df_desc.drop('Drug', 1)
+    if add_prefix:
+        df2 = df2.add_prefix('dragon7.')
```
Comment (hyoo) [inline Pilot1/Uno/uno_data.py]: check prefix here as well

**Comment by aclyde11** [comment]
fixed that change @hyoo

**Pull request merged**